CONTRIBUTING.md

```diff
@@ -2,126 +2,98 @@
 
 We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
 
+## Ways you can contribute
 
-## Ways you can contribute:
-   - [Report Issues](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#report-issues)
-   - [Submit Pull Requests](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#submit-pull-requests)
-   - [Fix or Improve Documentation](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#fix-or-improve-documentation) 
-   - [Join Our Community Meetings](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#join-our-community-meetings) 
+- [Report Issues](#report-issues)
+- [Submit Pull Requests](#submit-pull-requests)
+- [Fix or Improve Documentation](#fix-or-improve-documentation)
+- [Join Our Community Meetings](#join-our-community-meetings)
 
 ### Report issues
-   - Report potential bugs
-   - Request a feature
-   - Request a sample policy
 
-### Submit Pull Requests
-#### Setup local development environments 
--  Please refer to [Running in development mode](https://github.com/kyverno/kyverno/wiki/Running-in-development-mode) for local setup.
+Issues to Kyverno help improve the project in multiple ways including the following:
 
-####  Submit a PR for [open issues](https://github.com/kyverno/kyverno/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-### Fix or Improve Documentation
-   - [Kyverno Docs](https://github.com/kyverno/website)
-   #### Get started
+- Report potential bugs
+- Request a feature
+- Request a sample policy
 
-Head over to project repository on github and click the **"Fork"** button. With the forked copy, you can try new ideas and implement changes to the project.
+### Submit Pull Requests
 
- -  **Clone the repository to your device:**
+[Pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PRs) allow you to contribute back the changes you've made on your side enabling others in the community to benefit from your hard work. They are the main source by which all changes are made to this project and are a standard piece of git operational flows. Before you contribute, please take a moment to review and agree to abide by our community [Code of Conduct](/CODE_OF_CONDUCT.md).
```

Comment:
```suggestion
[Pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PRs) allow you to contribute back the changes you've made on your side enabling others in the community to benefit from your hard work. They are the main source by which all changes are made to this project and are a standard piece of GitHub operational flows. Before you contribute, please take a moment to review and agree to abide by our community [Code of Conduct](/CODE_OF_CONDUCT.md).
```

Pull Requests by the way they are implemented in GitHub are specific to GitHub - it's not a core feature of git.